### PR TITLE
separates out CLI-specific settings from IDE-specific

### PR DESCRIPTION
**Pull request by dmalan**
This way, we can include `cli.sh` in CS50 CLI, without pulling in IDE-specific settings.

### CODE REVIEW

**Comment by kzidane** [comment]
Rebased.

File: files/etc/profile.d/cli.sh [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+export PYENV_ROOT=/opt/pyenv
+export PATH="$PYENV_ROOT"/shims:"$PYENV_ROOT"/bin:"$PATH"
+alias pip="pip3"
+alias pylint="pylint3"
+alias python="python3"
+export PYTHONDONTWRITEBYTECODE="1"
+
+# ruby
+export RBENV_ROOT=/opt/rbenv
+export PATH="$RBENV_ROOT"/shims:"$RBENV_ROOT"/bin:"$PATH"
```
Comment (kzidane) [inline files/etc/profile.d/cli.sh]: The IDE already has `rvm` installed. Should we use this instead?

File: files/etc/profile.d/cli.sh [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+export LC_ALL="en_US.UTF-8"
+export LC_CTYPE="en_US.UTF-8"
+export NODE_ENV="dev"
+export PATH=/opt/cs50/bin:/usr/sbin:/usr/local/sbin:/usr/local/bin:/usr/bin:/sbin:/bin
+export PROMPT_COMMAND='__git_ps1 "\w/" " \\\$ "'
+export PYTHONDONTWRITEBYTECODE=1
+export TERM=xterm
+
+# apt-get
+export DEBIAN_FRONTEND=noninteractive
```
Comment (kzidane) [inline files/etc/profile.d/cli.sh]: Why do we want this to be always `noninteractive`?

File: files/etc/profile.d/cli.sh (hunk 1 above)
Comment (dmalan) [reply files/etc/profile.d/cli.sh]: Did Cloud9 install that?

**Comment by dmalan** [comment]
Made a lot of changes since, so closing for now!

File: files/etc/profile.d/cli.sh (hunk 1 above)
Comment (kzidane) [reply files/etc/profile.d/cli.sh]: Yes. It's part of their base image https://github.com/c9/templates/blob/master/workspace/Dockerfile#L91.

**Pull request closed without merging**